src/content/guides/development.md

```diff
@@ -420,4 +420,4 @@ To disable this feature in some common editors, see the list below:
 
 ## Conclusion
 
-Now that you've learned how to automatically compile your code and run a simple development server, you can check out the next guide, which will cover [Hot Module Replacement](/guides/hot-module-replacement).
+Now that you've learned how to automatically compile your code and run a simple development server, you can check out the next guide, which will cover [Code Splitting](/guides/code-splitting/).
```

Comment:
Logically, next is HMR, as this is development guide, despite the sidebar order.

Reply:
Yes, the HMR guide is mentioned and linked several times in this guide.

So, should the order in the sidebar change or should it be left as is and change the link to Code Splitting?

Reply:
its logical next, not order next

We could change this to further reading similar as we have here: 
https://webpack.js.org/guides/hot-module-replacement/

![Annotation 2019-08-12 210225](https://user-images.githubusercontent.com/10549495/62886823-95ca4e00-bd44-11e9-9583-a14f63b329aa.png)

But i don't think this is really necessary unless there are people raising issues with it

Reply:
But shouldn't the guides be read in the order in which they are in the sidebar? I believe that's how most people read them. So, if the intent is to make people read about HMR next, shouldn't that be next in the sidebar?

Reply:
All right, I will set the next guide link back to HMR. However, I do suggest that the order in the sidebar is changed so that the HMR guide comes after the *Development* guide for the reason I mentioned in my previous comment.